modules/nodes-nodes-resources-configuring-about.adoc

```diff
@@ -29,65 +28,35 @@ introduction of allocatable resources.
 An allocated amount of a resource is computed based on the following formula:
 
 ----
-[Allocatable] = [Node Capacity] - [kube-reserved] - [system-reserved] - [Hard-Eviction-Thresholds]
+[Allocatable] = [Node Capacity] - [system-reserved] - [Hard-Eviction-Thresholds]
 ----
 
 [NOTE]
 ====
-The withholding of `Hard-Eviction-Thresholds` from allocatable is a change in behavior to improve
-system reliability now that allocatable is enforced for end-user pods at the node level.
-The `experimental-allocatable-ignore-eviction` setting is available to preserve legacy behavior,
-but it will be deprecated in a future release.
+The withholding of `Hard-Eviction-Thresholds` from `Allocatable` improves system reliability because the value for `Allocatable` is enforced for pods at the node level.
 ====
 
-If `[Allocatable]` is negative, it is set to *0*.
+If `Allocatable` is negative, it is set to `0`.
 
-Each node reports system resources utilized by the container runtime and kubelet.
-To better aid your ability to configure `--system-reserved` and `--kube-reserved`,
-you can introspect corresponding node's resource usage using the node summary API,
-which is accessible at `/api/v1/nodes/<node>/proxy/stats/summary`.
+Each node reports the system resources that are used by the container runtime and `kubelet`. To simplify configuring `system-reserved`, view the resource use for the node by using the node summary API. The node summary is available at `/api/v1/nodes/<node>/proxy/stats/summary`.
```

Comment:
Probably don't need mark-up for kubelet.
s/confguring `system-reserved`/configuring the `system-reserved parameter ?